indico/modules/events/abstracts/templates/management/_abstract_list.html

```diff
@@ -29,6 +29,25 @@
     </td>
 {%- endmacro %}
 
+{% macro _render_abstract_state(abstract) %}
+    <div class="i-tag outline semantic-text {{ abstract.public_state.css_class }} state-badge">
+        {% if abstract.public_state.name == 'under_review' and abstract.can_judge(session.user) %}
+            {% trans count=abstract.reviews|length -%}
+                {{- count }} review
+            {%- pluralize -%}
+                {{- count }} reviews
+            {%- endtrans %}
+            ({% trans count=abstract.reviewed_for_tracks|length -%}
```

Comment:
I'd wrap after the `(` and then use `{%- trans` - like this it always looks like incorrect indentation before noticing the `(`